test/unit/controllers/CustomsNotificationControllerSpec.scala

```diff
@@ -60,32 +70,38 @@ class CustomsNotificationControllerSpec extends UnitSpec with Matchers with Mock
 
   private val emulatedServiceFailure = new EmulatedServiceFailure()
 
```

Comment:
should `val expectedRequestMetaData` be `private val expectedRequestMetaData` ?

Reply:
👍